README.md

```diff
@@ -44,45 +64,77 @@ a researcher to maintain a version-controlled R Markdown website:
+workflowr builds on several software tools including
+[git](http://cran.r-project.org), [pandoc](http://pandoc.org) and
+[knitr](yihui.name/knitr), **but you do not need to use any of these
+tools to get started with workflowr.** You only need to know how to
+code in [R](http://cran.r-project.org) and be familiar with
+[R Markdown](http://r4ds.had.co.nz/r-markdown.html). A [basic
+understanding of git](https://doi.org/10.1371/journal.pcbi.1004668)
+as well as the [UNIX command line](https://swcarpentry.github.io/shell-novice)
+is not essential, but helpful.
 
-* Install [Git][git] ([instructions][swc-git] from [Software Carpentry][swc])
+Here is a minimal set of steps to get you started with `workflowr`. If
+you are already using R and git, you may be able to skip some of these
+steps.
 
-* Create an account on [GitHub][gh]
+1. Install [R][r] ([instructions][swc-r] from [Software Carpentry][swc]).
 
-* Configure [Git][git] ([instructions][swc-git-config] from [Software
-Carpentry][swc]). Run the following in the Terminal, inserting your information:
-
-```bash
-git config --global user.name "Your Name"
-git config --global user.email "youremail@domain"
-```
-
-* Install the latest stable release of [workflowr][] from [GitHub][gh] using
-[devtools][]:
-
-```r
-# install.packages("devtools")
-devtools::install_github("jdblischak/workflowr", build_vignettes = TRUE)
-```
-
-* Work through the vignette [Getting started with workflowr][vig-start] to learn
-the basic design and functionality of a workflowr project (you can view all the
-available vignettes locally with `browseVignettes("workflowr")`)
-
-* Start your project with `wflow_start()` (if you have already started your
-project, please read the vignette [Migrating an existing project to use
-workflowr][vig-migrating] for advice on how to proceed)
-
-* Read through the vignette [Customize your research website][vig-custom]
+2. (Optional) Install [RStudio][rstudio] (workflowr takes advantages of some
+RStudio features, but RStudio is not required to use workflowr)
 
-* If you find any unexpected behavior or think of an additional feature that
-would be nice to have, please open an Issue [here][issues]. When writing your
-bug report or feature request, please note the version of workflowr you are
-using (which you can obtain by running `packageVersion("workflowr")`).
+3. Install [Git][git] ([instructions][swc-git] from [Software Carpentry][swc])
+
+4. Create an account on [GitHub][gh], or another git-based source code
+site such as [GitLab](http://gitlab.com). (In the instructions below
+we mostly assume Github, but workflowr should also work on other
+similar platforms such as [Gitlab](http://gitlab.com) and
+[Bitbucket](bitbucket.org).)
+
+    > **pc-comment-3:** I think there may be some additional setup
+    > steps to take here, some of which may not be strictly necessary,
+    > but helpful (e.g., SSH keys). Maybe refer to a Getting Started
```

Comment:
It has been several years since I've created a Github account and I seem to recall that there were more steps, but [maybe not](https://help.github.com/articles/set-up-git).